docs/advanced/features/invoices.mdx

```diff
@@ -95,7 +95,7 @@ Here is an example of a custom router that proxy a Magento remote url with
 additional checks:
 
 ```js
-import proxy from "express-http-proxy";
+import proxy from "server/core/proxy";
```

Comment:
Since we've backported the fix to all support branch I haven't documented any versions. If users aren't on the latest patch and complain, we'll ask to upgrade.